proto/storage/cve.proto

```diff
@@ -114,6 +121,7 @@ message CVEInfo {
   CVSSV2 cvss_v2 = 8;
   CVSSV3 cvss_v3 = 9;
   repeated Reference references = 10;
+  repeated CVSSScore cvss_scores = 11;
```

Comment:
In line 140 there's `repeated CVSSScore cvss_metrics = 11;`
I wonder if we need to keep the naming consistent here.
it can be cvss_xxx in both places.